webapp/channels/src/components/admin_console/team_channel_settings/convert_confirm_modal.tsx

```diff
@@ -115,10 +128,11 @@ const messages = defineMessages({
     },
     toPublicMessage: {
         id: 'admin.team_channel_settings.convertConfirmModal.toPublicMessage',
-
-        // This eslint-disable comment can be removed once this component no longer uses FormattedMarkdownMessage
-        // eslint-disable-next-line formatjs/no-multiple-whitespaces
-        defaultMessage: 'When you convert **{displayName}** to a public channel, history and membership are preserved. Public channels are discoverable and can by joined by users on the system without invitation.  \n \nAre you sure you want to convert **{displayName}** to a public channel?',
+        defaultMessage: 'When you convert {displayName} to a public channel, history and membership are preserved. Public channels are discoverable and can be joined by users on the system without invitation.',
```

Comment:
`{displayName} ` was bold before and now its not, lets not change the structure

Reply:
same in other places as well

Reply:
![image](https://github.com/user-attachments/assets/826e4b15-9547-4fcf-8cfb-02052ff1559d)

It's still like that..

Reply:
Oh i see apologies. But can you make a screenshot to visually verify

Reply:
@M-ZubairAhmed how to start the server??

Reply:
**Before**:
![image](https://github.com/user-attachments/assets/2c4eba79-064d-4f8e-9bd6-c0e0430ed1a9)

![image](https://github.com/user-attachments/assets/1b1055c3-3a5c-41ba-a13d-a8447b1a8388)
<figcaption>not getting an option to convert a pvt channel to pub channel</figcaption>

**After**:
![image](https://github.com/user-attachments/assets/7ce70919-4282-4ab7-8093-51558b096761)

Reply:
@M-ZubairAhmed check it

Reply:
> not getting an option to convert a pvt channel to pub channel
Don't know why!

Reply:
awesome, please include these in the PR description as well for QA reference

Reply:
Already did! Thanks!!!